engines/ags/ags.cpp

```diff
@@ -386,7 +386,11 @@ SaveStateList AGSEngine::listSaves() const {
 }
 
 void AGSEngine::setGraphicsMode(size_t w, size_t h) {
+#ifdef SCUMM_LITTLE_ENDIAN
```

Comment:
Why this ifdef? It also applies on x86 targets.
It implies that this piece of code didn't work on x86 either.
Your previous PR used OPENGL here.

Reply:
Thank you for noticing. Applied the wrong (older) patch by accident. Amended now.